CONTRIBUTING.md

```diff
@@ -55,25 +55,67 @@ From the repository root.
 Before making a PR, you'll need to do two things to get CI passing:
 
 1. Generate schema files for the contracts.
```

Comment:
should we also make a note about running `cargo schema`?

Reply:
I think this is covered by the earlier section that mentions running `scripts/schema.sh` which does that under the hood.

Reply:
oh shit wait you're right the order is wrong..

Reply:
or wait no this is correct. This section is giving an overview, but the end result is just prompting the reader to run the `schema.sh` script.